### PR TITLE
[4.0] PLG_SYSTEM_LANGUAGEFILTER_XML_DESCRIPTION

### PR DESCRIPTION
**Pull request by brianteeman**
This PR removes two sentences from the plugin description as follows

1. This plugin is to be enabled only when the Language Switcher module is published.
2. If this plugin is activated, it is recommended to also publish the Administrator multilingual status module.

Unless it causes an error on the site I can't see the logic for sentence 1
The recommendation in sentence 2 is no longer valid as the module is automatically published if the plugin is enabled

### CODE REVIEW

File: administrator/language/en-GB/en-GB.plg_system_languagefilter.ini [hunk 1]
```diff
@@ -18,4 +18,5 @@ PLG_SYSTEM_LANGUAGEFILTER_OPTION_SESSION="Session"
 PLG_SYSTEM_LANGUAGEFILTER_OPTION_YEAR="Year"
 PLG_SYSTEM_LANGUAGEFILTER_PRIVACY_CAPABILITY_LANGUAGE_COOKIE="On a site which supports multiple languages, this plugin can be configured to set a cookie on the user's browser which remembers their language preference. This cookie is used to redirect users to their preferred language when visiting the site and creating a new session. The cookie's name is based on a randomly generated hash and therefore does not have a constant identifier."
 PLG_SYSTEM_LANGUAGEFILTER_SITE_LANGUAGE="Site Language"
-PLG_SYSTEM_LANGUAGEFILTER_XML_DESCRIPTION="This plugin filters the displayed content depending on language.<br><strong>This plugin is to be enabled only when the Language Switcher module is published.</strong><br>If this plugin is activated, it is recommended to also publish the Administrator multilingual status module."
+PLG_SYSTEM_LANGUAGEFILTER_XML_DESCRIPTION="This plugin filters the displayed content depending on language.""
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.plg_system_languagefilter.ini]:
```suggestion
PLG_SYSTEM_LANGUAGEFILTER_XML_DESCRIPTION="This plugin filters the displayed content depending on language."
```

File: administrator/language/en-GB/en-GB.plg_system_languagefilter.ini [hunk 2]
```diff
@@ -18,4 +18,5 @@ PLG_SYSTEM_LANGUAGEFILTER_OPTION_SESSION="Session"
...
+
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.plg_system_languagefilter.ini]:
```suggestion
```

**Comment by tonypartridge** [comment]
I have tested this item :white_check_mark: successfully on ae6ae6fcca723a72db95e9fb92e9c780dfd382ca<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24863">issues.joomla.org/tracker/joomla-cms/24863</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on ae6ae6fcca723a72db95e9fb92e9c780dfd382ca<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24863">issues.joomla.org/tracker/joomla-cms/24863</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24863">issues.joomla.org/tracker/joomla-cms/24863</a>.</sub>

**Comment by infograf768** [comment]
> The recommendation in sentence 2 is no longer valid as the module is automatically published if the plugin is enabled.

Not sure about this in new admin template

**Comment by brianteeman** [comment]
I am 100% certain

**Comment by tonypartridge** [comment]
Great! Your hugely better at English than me so I’ll go with your wording
:-)

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**